source/gui/__init__.py

```diff
@@ -737,13 +737,13 @@ def _appendHelpSubMenu(self, frame: MainFrame) -> None:
 		if not globalVars.appArgs.secure:
 			# Translators: The label of a menu item to open NVDA user guide.
 			item = self.helpMenu.Append(wx.ID_ANY, _("&User Guide"))
-			self.Bind(wx.EVT_MENU, lambda evt: os.startfile(getDocFilePath("userGuide.html")), item)
+			self.Bind(wx.EVT_MENU, lambda evt: self._openDocumentationFile("userGuide.html"), item)
```

Comment:
Can `evt` be changed to `_` to show it is unused?
```suggestion
			self.Bind(wx.EVT_MENU, lambda _: self._openDocumentationFile("userGuide.html"), item)
```

Reply:
I'd rather avoid to use "_" (underscore character) for unused variables in our code.

Indeed it is already used as a gettext translation function. IMO, for clarity, it's better not to use the same variable name for two distinctive usages, even if the scope is not the same.

And what if you want to use translated content in a lambda function taking unused arguments in the future?

Reply:
> And what if you want to use translated content in a lambda function taking unused arguments in the future?

You could still use `gettext` directly.

Nevertheless, that's a very good point. I'm personally not a huge fan of having named unused variables, but it seems there's no very good alternative here.

Reply:
Moreover here, naming it `evt` also conveys the fact that this function is passed the event (even if this event is not used).

If you are concerned by the fact that we do not indicate that the variable is unused, I may name it `unusedEvt` instead, but that's a bit ugly and probably not necessary for a so short function body.

Reply:
No, I think having it named `evt` is best here. Sorry for the fuss